src/tests/integration/test-data/end-to-end-prove-all

```diff
@@ -1,14 +1,25 @@
 CounterTest.test_Increment()
+ForgetBranchTest.test_forgetBranch(uint256)
```

Comment:
I only added this new test, rest of the changes in this file come from alphabetical sorting.